syntax = "proto3";

package auth;

service AuthService {
  rpc RegisterUser(AuthRequest) returns (AuthResponse);
  rpc LoginUser(AuthRequest) returns (AuthResponse);
}

message AuthRequest {
  string username = 1;
  string password = 2;
}

message AuthResponse {
  bool success = 1;
  string message = 2;
}
